src/components/Grid/Grid.jsx

```diff
@@ -0,0 +1,124 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+
+const boundGridClassNames = bindClassNames('Grid');
+const boundCellClassNames = bindClassNames('Cell');
+
+/**
+ * {"categories": ["layout", "grid"]}
+ *
+ * This component is designed to be used in Composits as a layout tool.
+ * You can use the Grid components themselves or create your own components using the Grid styles from Grid.less.
+ * Please see examples/CSS-Grid.jsx for more information.
+ */
+
+const {
+	object,
+	string,
+	node,
+} = React.PropTypes;
+
+function propsToClass (props) {
+	const classNameProps = _.reduce(props, (memo, value, key)=>{
+			if(_.startsWith(key, 'is-')){
+				return memo.concat([key]);
+			}
+			return memo;
+		}, []);
+
+	if(_.isEmpty(classNameProps)){
+		return null
+	}
+
+	return classNameProps;
+}
+
+const Grid = React.createClass({
```

Comment:
we should be using the `createLucidComponentDefinition` within `React.createClass` which will also let u declare `Cell` a child component of `Grid`